### PR TITLE
[prod] 28.0.14 RC1, 30.0.4 RC1, escape route for 28.0.13 installations

### PR DESCRIPTION
**Pull request by blizzz**
config.php tests is failing knowlingly due to the added upgrade path away from 28.0.13 instances

### CODE REVIEW

File: config/config.php [hunk 1]
```diff
@@ -710,34 +758,34 @@
...
 			'100' => [
-				'latest' => '28.0.13 RC1',
-				'internalVersion' => '28.0.13.0',
-				'downloadUrl' => 'https://download.nextcloud.com/server/prereleases/nextcloud-28.0.13rc1.zip',
+				'latest' => '28.0.14 RC1',
+				'internalVersion' => '28.0.14.0',
+				'downloadUrl' => 'https://download.nextcloud.com/server/prereleases/nextcloud-28.0.14rc1.zip',
 				'web' => 'https://docs.nextcloud.com/server/28/admin_manual/maintenance/upgrade.html',
-				'eol' => false,
+				'eol' => true,
```
Comment (nickvergessen) [inline config/config.php:765]: false?

File: config/config.php [hunk 2]
```diff
@@ -710,34 +758,34 @@
...
```
Comment (nickvergessen) [inline config/config.php:781]: same here?

File: config/config.php [hunk 3]
```diff
@@ -678,21 +694,53 @@
...
+CrfaBIp6zQFLh+r/v0IjhQ==',
+			],
+		],
+		'28.0.14.0' => [
+			'100' => [
+				'latest' => '29.0.10',
+				'internalVersion' => '29.0.10.1',
+				'downloadUrl' => 'https://download.nextcloud.com/server/releases/nextcloud-29.0.10.zip',
+				'web' => 'https://docs.nextcloud.com/server/29/admin_manual/maintenance/upgrade.html',
+				'eol' => false,
```
Comment (nickvergessen) [inline config/config.php:733]: it is false here, that's why I'm asking

File: config/config.php (hunk 1 above)
Comment (blizzz) [reply config/config.php:765]: why?

File: config/config.php (hunk 3 above)
Comment (blizzz) [reply config/config.php:733]: that's 29

Comment (nickvergessen) [reply config/config.php:733]: ah, read it the wrong way...

**Review by nickvergessen — APPROVED** [review]

**Pull request merged**